src/index.js

```diff
@@ -126,6 +128,10 @@ export default function Loadable<Props: {}, Err: Error>(opts: Options) {
     render() {
       let { pastDelay, error, Component } = this.state;
 
+      if (!isWebpack && serverSideRequirePath) {
```

Comment:
@smashercosmo With @thejameskyle's wonderful work on #33 in place, the implementation gets much simpler. I dropped an MWE at <https://github.com/phyllisstein/react-loadable-mwe> anyway, in case it's at all helpful.